app/javascript/components/explore/FacetFilteringModal.jsx

```diff
@@ -0,0 +1,52 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faInfoCircle } from '@fortawesome/free-solid-svg-icons'
+import React, { useState } from 'react'
+import Modal from 'react-bootstrap/lib/Modal'
+import { closeModal } from '~/components/search/controls/SearchPanel'
+
+/** Show modal box with information on differential expression in SCP */
+export default function FacetFilteringModal() {
+  const [showCellFacetModal, setShowCellFacetModall] = useState(false)
+  const cellFacetModalContent = (
+    <div>
+      <p> Cell filtering provides the ability to view cell annotations within the context of other annotations.
+        You can visualize the expression of particular cells in a subset of the dataset through the facet filters.
```

Comment:
I hadn't fully realized that this will extend to expression-based visualizations, but it does totally work!  Very cool.